media-libs/embree/metadata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<!-- maintainer-needed -->
+	<longdescription>
+		Intel® Embree is a collection of high-performance ray tracing kernels,
+		developed at Intel. The target users of Intel® Embree are graphics
+		application engineers who want to improve the performance of their
+		photo-realistic rendering application by leveraging Embree's
+		performance-optimized ray tracing kernels.
+	</longdescription>
+	<use>
+		<flag restrict="&gt;=media-libs/embree-3.12.2" name="compact-polys">
+			Enables double indexed poly layout.
+		</flag>
+		<flag restrict="&gt;=media-libs/embree-3.12.2" name="ssp">
+			When enabled, Embree compiles with stack protection against return address overrides.
```

Comment:
Do you have experience on the performance impact of this flag? If so, and if it has a reasonable impact, what do you think about informing the users about this?
Especially with a resource intensive application like blender, they may want to consider whether to enable the flag or not and we give them the power to make an informed decision.